tensorflow/tools/docker/Dockerfile.devel

```diff
@@ -38,6 +38,8 @@ RUN pip --no-cache-dir install \
         && \
     python -m ipykernel.kernelspec
 
+# RUN ln -s /usr/bin/python3 /usr/bin/python#
```

Comment:
Is it commented out on purpose?

Reply:
Seems like expected according to Amit's reply in https://github.com/tensorflow/tensorflow/pull/18395#pullrequestreview-111003879